Allow to set power savings mode of BH1750 light level sensor

nohum opened the pull request:
According to the sources of the Arduino BH1750 library ([see here](https://github.com/mysensors/MySensorsArduinoExamples/blob/master/libraries/BH1750/BH1750.h#L63)), the sensor is started by default in the high resolution mode without any energy savings activated.  

The underlying library allows to change this setting to allow for power savings, but fortunately this is not exposed by NodeManager. This pull request changes this and adds a possibility to modify this setting, e.g. by calling:

```
int light = nodeManager.registerSensor(SENSOR_BH1750);

SensorBH1750* lightSensor = ((SensorBH1750*) nodeManager.getSensor(light));
lightSensor->setMode(BH1750_ONE_TIME_LOW_RES_MODE);
```

user2684 commented:
Thanks for the contribution. I've noticed though that you started your development from v1.5.1 and not from the development branch which makes merging it very difficult. Since the changes are not too many I'd recommend to start from the development branch. Also, if you think makes sense to expose setMode() through the remote API, you can add it to the onProcess() function of the sensor. Let me know if you need any help with it. Many thanks!

nohum commented:
Hello! Thanks for your hints. I changed the base, exposed `setMode` via the remote API and while i was at doing that, i updated the docs in the README.md too ;)

user2684 commented:
Fantastic thanks! I was planning to post-pone any enhancement to the next release but this looks very clean to me so it will be part of the upcoming v1.6 and merged into development during the weekend. I've also added a corresponding issues for this #197
Many thanks!

user2684 commented on NodeManager.h:
```diff
@@ -972,6 +972,8 @@ class SensorDs18b20: public Sensor {
 class SensorBH1750: public Sensor {
   public:
     SensorBH1750(NodeManager* node_manager, int child_id);
+    // set sensor reading mode, e.g. BH1750_ONE_TIME_HIGH_RES_MODE
```
Would you mind both here and in the documentation making this comment looking like the following:
`// [101] set sensor reading mode, e.g. BH1750_ONE_TIME_HIGH_RES_MODE`
So to make explicit the id to use for the remote api like the other functions. Thanks!

nohum commented:
Done

user2684 commented:
Fantastic thanks a lot for the contribution, merged into development :)
